content/docs/configuration/annotations.md

```diff
@@ -0,0 +1,326 @@
+
+this annotation allows you to configure `spec.dnsNames` field for
+the Certificate to be generated.
+Supports comma-separated values e.g. "example.com,example.org"
+
+## cert-manager.io/certificate-name
+- [CertificateRequest](../usage/certificaterequest.md)
+
+name of the related certificate.
+
+## cert-manager.io/certificate-revision
+- [CertificateRequest](../usage/certificaterequest.md)
+
+the iteration the certificate request.
+
+## cert-manager.io/cluster-issuer
+- [Ingress](../usage/ingress.md)
+- [Gateway](../usage/gateway.md)
+
+the name of a cert-manager.io ClusterIssuer that should issue the required certificate.
+
+## cert-manager.io/common-name
+- [Ingress](../usage/ingress.md)
+- [Gateway](../usage/gateway.md)
+
+this annotation allows you to configure `spec.commonName` for the Certificate
+to be generated.
+
+## cert-manager.io/duration
+- [Ingress](../usage/ingress.md)
+- [Gateway](../usage/gateway.md)
+
+this annotation allows you to configure `spec.duration` field for the
+Certificate to be generated.
+
+## cert-manager.io/email-sans
+- [Ingress](../usage/ingress.md)
+- [Gateway](../usage/gateway.md)
+
+this annotation allows you to configure `spec.emailAddresses` field for
+the Certificate to be generated.
+Supports comma-separated values e.g. "me@example.com,you@example.com"
+
+## cert-manager.io/ip-sans
+- [Ingress](../usage/ingress.md)
+- [Gateway](../usage/gateway.md)
+
+this annotation allows you to configure `spec.ipAddresses` field for
+the Certificate to be generated.
+Supports comma-separated values e.g. "198.51.100.1,198.51.100.2"
+
+## cert-manager.io/issuer-group
+- [Ingress](../usage/ingress.md)
+- [Gateway](../usage/gateway.md)
+
+the API group of the external issuer controller, for example
+`awspca.cert-manager.io`. This is only necessary for out-of-tree issuers.
+
+## cert-manager.io/issuer-kind
+- [Ingress](../usage/ingress.md)
+- [Gateway](../usage/gateway.md)
+
+the kind of the external issuer resource, for example `AWSPCAIssuer`. This
+is only necessary for out-of-tree issuers.
+
+## cert-manager.io/issuer
+- [Ingress](../usage/ingress.md)
+- [Gateway](../usage/gateway.md)
+
+the name of the issuer that should issue the required certificate.
+
+## cert-manager.io/issue-temporary-certificate
+- [Certificate](../usage/certificate.md)
+
+cause a [temporary
+certificate](../usage/certificate.md#temporary-certificates-whilst-issuing) to
+be set on the resulting `Secret` until the final signed certificate has been
+returned.
+This is useful for keeping compatibility with the `ingress-gce` component.
+
+## cert-manager.io/inject-apiserver-ca
+
+cause the `cainjector` to inject the **CA certificate** for the Kubernetes apiserver into the resource.
+
+## cert-manager.io/inject-ca-from
+
+cause the `cainjector` to inject a certificate with **CA certificate**. ??
+
+## cert-manager.io/inject-ca-from-secret
+
+cause the `cainjector` to inject a **CA Certificate** from a secret.
```

Comment:
I guess this stuff is documented in content/docs/concepts/ca-injector.md and I'll have to copy some content over and add cross links